docs/integrations/amazon-aws/lambda.md

```diff
@@ -168,11 +168,14 @@ To configure a CloudTrail Source, perform these steps:
 
 ### Collect Amazon CloudWatch Metrics
 
-To collect Amazon CloudWatch Metrics, see [Amazon CloudWatch Source For Metrics.](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
-
-AWS Namespace tag to filter in source for Lambda will be - **AWS/Lambda**
-
-* **Metadata: **Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account field”.
+* Sumo Logic supports collecting metrics using two source types
+    * Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
+       or
+    * Configure an [AWS Kinesis Firehose for Metrics Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Amazon-Web-Services/AWS_Kinesis_Firehose_for_Metrics_Source) (Recommended)
+:::note
+Namespace for **Amazon Lambda** Service is **AWS/Lambda**.
+:::
+    * **Metadata**: Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account” field.
```

Comment:
```suggestion
    * **Metadata**. Add an **account** field to the source and assign it a value that is a friendly name/alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. The **account** field allows you to query metrics.
```

Reply:
Can you please check how the "account" field is written in UI? If it's in uppercase, then we should change here as well.

Reply:
Its mentioned as "account" itself.

Reply:
@nmohantyctr Thanks for the confirmation.